crates/extrinsics/src/integration_tests.rs

```diff
@@ -485,13 +490,14 @@ async fn api_build_upload_instantiate_call() {
         .unwrap();
     let instantiate_result = instantiate.instantiate(None).await;
     assert!(instantiate_result.is_ok(), "instantiate code failed");
-    let instantiate_result = instantiate_result.unwrap();
+    let instantiate_result: InstantiateExecResult<DefaultConfig> =
+        instantiate_result.unwrap();
     let contract_account = instantiate_result.contract_address.to_string();
     assert_eq!(48, contract_account.len(), "{contract_account:?}");
 
     // call the contract
     // the value should be true
-    let call = CallCommandBuilder::default()
+    let call = CallCommandBuilder::<DefaultConfig, DefaultEnvironment, _, _>::default()
```

Comment:
It would be nice not to have to do this turbofish, perhaps the generic params could default to the default types here and then you can use a builder method `e.g. `.with_env::<MyConfig, MyEnvironment>`